docs/userguides/plugins.md

```diff
@@ -118,3 +118,21 @@ CLI arguments are decoded and passed in to the application with `click`.
 CLI plugins will use the plugin registration process defined above. The
 CLI plugins should use the `click` library in order to be able to supply
 arguments from the CLI.
+
+## Logging
+
+Use Ape's logger in your plugins or scripts by importing it from the `ape.logging` module or 
+by using it off the CLI context (from using the `@ape_cli_context` decorator).
+  
+* **ape_cli_context:**
+  
+Use it from the @ape_cli_context decorator:
+```bash
+from ape.logging import logger
+
+logger.info("This is a log message")
```

Comment:
Might be worth breaking these into two separate code-blocks each with their own boldened header, like this comment: https://github.com/ApeWorX/ape/pull/341/files#r773420374